libpkg/pkg_create.c

```diff
@@ -95,6 +95,12 @@ pkg_create_from_dir(struct pkg *pkg, const char *root,
 			return (EPKG_FATAL);
 		}
 
+		if (!S_ISREG(st.st_mode)) {
```

Comment:
This should also accept symlinks! (which is what break the test suite)

Reply:
Well, that's embarrassingly silly.  Fixed.